src/app/helpers/config/configuration.js

```diff
@@ -0,0 +1,15 @@
+
+let configData = {}
+
+__get = function(configKey){
+    return configData[configKey]
+}
+
+__set = function(configKey,configValue){
+    configData[configKey] = configValue
+}
+
+module.exports = {
+    get: __get,
```

Comment:
where is the get and set used?

Reply:
The exported get and set methods were used in confighelper.js file to set and get configs